git/client_test.go

```diff
@@ -471,22 +471,32 @@ func TestClientCommits(t *testing.T) {
 		{
 			name:        "get commits",
 			cmdStdout:   "6a6872b918c601a0e730710ad8473938a7516d30,testing testability test",
-			wantCmdArgs: `path/to/git -c log.ShowSignature=false log --pretty=format:%H,%s --cherry SHA1...SHA2`,
+			wantCmdArgs: `path/to/git -c log.ShowSignature=false log --pretty=format:%H,%s,%b --cherry SHA1...SHA2`,
 			wantCommits: []*Commit{{
 				Sha:   "6a6872b918c601a0e730710ad8473938a7516d30",
 				Title: "testing testability test",
 			}},
 		},
+		{
+			name:        "get commits with body",
+			cmdStdout:   "6a6872b918c601a0e730710ad8473938a7516d30,testing testability test,This is the body",
```

Comment:
🤔 **What is a more complicated example that will ensure the multi-line body use case works as expected?**